_posts/blog/2019-08-19-Ruslan's-Final-Report.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: GSoC Final Report by Ruslan 
+categories: blog
+excerpt: work product submission on the scona project during GSoC program
+image:
+  feature:
+link:
+date: 2019-08-19
+modified:
+share: true
+author: ruslan_yermakov
+---
+
+## TL;DR - GSoC Summary
+
+> **Student**: Ruslan Yermakov  
+**Mentor**: Dr Kirstie Whitaker    
+**Organization**: [INCF](https://www.incf.org/gsoc2019/projectlist)    
+**Project**: In the course of the program I was working on the open-source project [**scona**](https://github.com/WhitakerLab/scona) - a toolkit to perform brain network analyses. 
+The goal of my project was to produce publication-ready brain network analysis results and visualizations from the command line. 
+
+In this blog I am sharing the achieved deliverables and results with the community.
+
+## The scope of the project
+
+Neuroscientists and neuroimaging researches perform structural network analyses on the MRI data.
+Scona is designed to imrove this experience and to help researchers run brain analyses quickly and reliably in order to capture many aspects of brain structure and function.
+
+My project is mainly focused on the visualisation part that is now included in the scona package.
+  
+Achieved deliverables:
+- implementation of visualization module to plot MRI data and the results of structural covariance brain network analyses;  
+- development of visualisation functions which allows users to produce publication-ready plots based on the results of the performed analysis for publication in a scientific journal;  
+- writing documentation and tutorials to help users better understand how to use the package;
+- fixing bugs and issues in the project; 
+
+
+#### Links to my contributions
+
+There are a bunch of [issues](https://github.com/WhitakerLab/scona/issues) within the project, which were used to discuss new features, to find solutions for challenges and to communicate the results. 
+You can easily search for issues by filtering using label “visualisations” and/or “enhancement”.
+
+Here are the links to my merged Pull Requests that cover all the work:
+1) [Implementation of visualisation tools #121](https://github.com/WhitakerLab/scona/pull/121)
+2) [Implementation of anatomical network visualisations #140](https://github.com/WhitakerLab/scona/pull/140)
+3) [Add Nilearn plotting functionality to scona #145](https://github.com/WhitakerLab/scona/pull/145)
+4) [TODO - open new one here new opened]()
+
+<figure class="half">
```

Comment:
I think this figure could go under another heading called something like "Some beautiful visualisations" (you can change that if you don't fancy using "beautiful" in the heading!) and then adding some text to say what those example visualisations are actually showing you.

Just one or two sentences for each picture in one or two paragraphs explaining why people would want to understand the brain.

If you give it a try I can edit any of the details, so don't worry about getting everything exactly right. But you do know a lot about brain network analyses now - the blog should showcase that!

Reply:
I will surely try, but I believe I will need your help to edit this part :)

And in answer to "why people would want to understand the brain", I can go really philosophic, e.g. like if we put the map (connectome) of neuron connections (trillions) inside the computer, will this make a computer think like a real human?

unraveling the mysteries around the brain (as it is the most secret part of the human body), investigating brain ( read "human") capabilities, finding patterns in brain and discovering the nature and reasons of human behavior, thoughts, feelings... what causes mental disorders, the process of aging(?) and so on and so forth...

Reply:
You did a great job @wingedRuslan!! Well done!